jetbrains-core/src/software/aws/toolkits/jetbrains/core/explorer/nodes/AwsExplorerServiceNode.kt

```diff
@@ -0,0 +1,12 @@
+// Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package software.aws.toolkits.jetbrains.core.explorer.nodes
+
+import com.intellij.openapi.project.Project
+
+interface AwsExplorerServiceNode {
+    val serviceId: String
```

Comment:
eg. - we can push `serviceId` and `displayName` into the `AwsExplorerServiceRootNode`

Reply:
Great points, let me try and refactor these.

Reply:
The Extension Point needs to point to an interface and for now should not be included into the AwsExplorerServiceRootNode class.

Reply:
This seems reasonable.